apps/docs/libraries/ts/ratelimit.mdx

```diff
@@ -93,23 +93,24 @@ By default the SDK will reject a request if it hasn't received a response from u
 
 The SDK captures most errors and handles them on its own, but we also encourage you to add a `onError` handler to configure what happens in case something goes wrong.
 
+Both `fallback` property of the `timeout` config and `onError` config are callback functions. They receive the original request identifier as one of their parameters, which you can use to determine whether to reject the request.
 
 ```ts
 import { Ratelimit } from "@unkey/ratelimit"
 
 // In this example we decide to let requests pass, in case something goes wrong.
 // But you can of course also reject them if you want.
-const fallback = { success: true, limit: 0, reset: 0, remaining: 0 }
+const fallback = (identifier: string) => ({ success: true, limit: 0, reset: 0, remaining: 0 })
 
 const unkey = new Ratelimit({
   // ... standard stuff
   timeout: {
     ms: 3000, // only wait 3s at most before returning the fallback
     fallback
```

Comment:
I believe this fallback value would not cause type issues because it expects a `RatelimitResponse` rather than `(string)=>RatelimitResponse`

Reply:
Sorry but I don't understand, I've already updated the type in [this](https://github.com/unkeyed/unkey/pull/1998/files#diff-8ade29a729b9ecba3fffe19ac2e7ac3529e9a7fb869a5b6287d546f81243f0a6R75) line of code. I'm also going to follow your suggestion in the other comment to support async fallback